avro/test_data.go

```diff
@@ -53,6 +53,61 @@ var testNestedArraySchema = `{
 }
 `
 
+var nestedObjectSchema = `{
+  "type": "record",
+	"name": "nested-object-example",
+	"fields": [
+	  {
+		  "name": "team",
+			"type": "string"
+	  },
+		{
+		  "name": "footballer",
+			"type": {
+			  "name": "footballer-name",
+			  "type": "record",
+				"fields": [
+					{
+			  		"name": "surname",
+						"type": "string",
+            "default": ""
```

Comment:
spacing

Reply:
I saw this and I swear I used go fmt........ :(

Reply:
Just realised its in a string so it won't format it, I have to do it manually :(